project-base/phpstan.neon

```diff
@@ -1,5 +0,0 @@
-parameters:
```

Comment:
the config is used here - https://github.com/shopsys/shopsys/blob/487132e2156d1b701a1eceb5ba079e136172cf5a/project-base/build.xml#L26

and then here - 
https://github.com/shopsys/shopsys/blob/487132e2156d1b701a1eceb5ba079e136172cf5a/project-base/build-dev.xml#L320

but if I get it correctly, it is unnecessary as it does not configure anything. If I am right, please remove the usages mentioned above as well.

Reply:
I would just add that it's used when you download and use `shopsys/project-base` instead of the `shopsys/shopsys` monorepository